### ISSUE TITLE
NDK events don't include breadcrumbs 

### ISSUE THREAD

**Issue by eakoli** [getsentry/sentry-android#518]
_Platform:_
- [ ]  Android -> If yes, which Device API (and compileSdkVersion/targetSdkVersion/Build tools) version?
- [ ]  Java -> If yes, which Java (and sourceCompatibility/targetCompatibility) version?
- [ ]  Kotlin -> If yes, which Kotlin (and jvmTarget) version?
- [ ]  NDK -> If yes, which NDK/CMake version?
- [ ]  React-Native -> If yes, which version?

_IDE:_
- [ ]  Android Studio -> If yes, which version?
- [ ]  IntelliJ -> If yes, which version?
- [ ]  Other -> If yes, which one?

_Build system:_
- [ ]  Gradle -> If yes, which version?
- [ ]  Buck -> If yes, which version?
- [ ]  Bazel -> If yes, which version?
- [ ]  Other -> If yes, which one?

_Android Gradle Plugin:_
- [ ]  Yes -> If yes, which version?
- [ ]  No

_Sentry Android Gradle Plugin:_
- [ ]  Yes -> If yes, which version?
- [ ]  No

_Proguard/R8:_
- [ ]  Enabled
- [ ]  Disabled

_sentry-android installed with:_
- [ ] JCenter
- [ ] Bintray
- [ ] Maven Central
- [ ] Manually

The version of sentry-android:
**2.x.x**

---
I have the following issue:

Crash/events generated from the NDK later (sentry-native) don't include any of the breadcrumbs generated from the android side.

Is there a plan to marry the two frameworks so that regardless of which side triggers the event they get comparable events and associated data? 

The description goes here ...

**Steps to reproduce:**
- Step

**Actual result:**
- Actual

**Expected result:**
- Result

**Comment by marandaneto** [comment]
@eakoli could you please fulfill the bug template? including the version of the SDK etc.

**Comment by eakoli** [comment]
@marandaneto updated, but the core issue seems to be that there is no data bridge between the java and native side. they run as separate systems. The only 'integration' is that the native side uses the java side as the transport mechanism to the server.

**Comment by marandaneto** [comment]
that's true, we still don't have the full sync. of Scopes, similar issue #174

this should come in the future though, I'll keep it open as this is specifically for breadcrumbs, so people can upvote and we could prioritize better.

we understand that the whole Scope (breadcrumbs, user, tags, extra...) should be synced on both sides (Android + NDK), thanks for reporting this.
